### PR TITLE
Updates for 2.1/3.1 servicing 

### PR DESCRIPTION
**Pull request by mthalman**
Related to #1921

### CODE REVIEW

File: README.sdk.preview.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ The [.NET Docker samples](https://github.com/dotnet/dotnet-docker/blob/master/sa
 Tags | Dockerfile | OS Version
 -----------| -------------| -------------
 5.0.100-preview.4-buster-slim, 5.0-buster-slim, 5.0.100-preview.4, 5.0, latest | [Dockerfile](https://github.com/dotnet/dotnet-docker/blob/master/5.0/sdk/buster-slim/amd64/Dockerfile) | Debian 10
-5.0.100-preview.4-alpine3.11, 5.0-alpine3.11, 5.0.100-preview.4-alpine, 5.0-alpine | [Dockerfile](https://github.com/dotnet/dotnet-docker/blob/master/5.0/sdk/alpine3.11/amd64/Dockerfile) | Alpine 3.11
+5.0.100-preview.4-alpine3.11, 5.0-alpine3.11 | [Dockerfile](https://github.com/dotnet/dotnet-docker/blob/master/5.0/sdk/alpine3.11/amd64/Dockerfile) | Alpine 3.11
```
Comment (MichaelSimons) [inline README.sdk.preview.md:65]: I see the floating tags were removed from all of the 5.0 images.  This doesn't seem correct until the next 5.0 release.

Comment (mthalman) [reply README.sdk.preview.md]: Fixed

File: manifest.json [hunk 2]
```diff
@@ -2830,7 +2842,9 @@
               "osVersion": "alpine3.11",
               "tags": {
                 "$(5.0-SdkVersion)-alpine3.11": {},
-                "5.0-alpine3.11": {}
+                "5.0-alpine3.11": {},
+                "$(5.0-RuntimeVersion)-alpine": {},
```
Comment (MichaelSimons) [inline manifest.json:2846]:
```suggestion
                "$(5.0-SdkVersion)-alpine": {},
```

Comment (mthalman) [reply manifest.json:2846]: Good catch

File: 3.1/runtime/alpine3.12/amd64/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,12 @@
+ARG REPO=mcr.microsoft.com/dotnet/core/runtime-deps
+FROM $REPO:3.1-alpine3.11
```
Comment (MichaelSimons) [inline 3.1/runtime/alpine3.12/amd64/Dockerfile:2]:
```suggestion
FROM $REPO:3.1-alpine3.12
```

File: 3.1/runtime/alpine3.12/arm64v8/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,12 @@
+ARG REPO=mcr.microsoft.com/dotnet/core/runtime-deps
+FROM $REPO:3.1-alpine3.11-arm64v8
```
Comment (MichaelSimons) [inline 3.1/runtime/alpine3.12/arm64v8/Dockerfile:2]:
```suggestion
FROM $REPO:3.1-alpine3.12-arm64v8
```

File: 3.1/sdk/alpine3.12/amd64/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    # Disable the invariant mode (set in base image)
+    DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=false \
+    # Enable correct mode for dotnet watch (only mode supported in a container)
+    DOTNET_USE_POLLING_FILE_WATCHER=true \
+    LC_ALL=en_US.UTF-8 \
+    LANG=en_US.UTF-8 \
+    # Skip extraction of XML docs - generally not useful within an image/container - helps performance
+    NUGET_XMLDOC_MODE=skip \
+    # PowerShell telemetry for docker image usage
+    POWERSHELL_DISTRIBUTION_CHANNEL=PSDocker-DotnetCoreSDK-Alpine-3.11
```
Comment (MichaelSimons) [inline 3.1/sdk/alpine3.12/amd64/Dockerfile:16]:
```suggestion
    POWERSHELL_DISTRIBUTION_CHANNEL=PSDocker-DotnetCoreSDK-Alpine-3.12
```

**Pull request merged**